dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/HvdcVSC/BaseControls/ACVoltageControl/QRefQU.mo

```diff
@@ -37,48 +37,46 @@ model QRefQU "Function that calculates QRef for the Q mode and the U mode depend
     Placement(visible = true, transformation(origin = {110, 80}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {110, -80}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
```

Comment:
Well, this one was necessary for a constant initialization. Moreover, I have rearranged the diagram.

Reply:
Why was it necessary ? 
Also, the diagram rearrangement is done in Mehdi's PR, thanks

Reply:
Modelica.Blocks.Nonlinear.SlewRateLimiter slewRateLimiter(Rising = SlopeURefPu, y(start = U0Pu + Lambda * Q0Pu))

y(start = ...) guarantees the constant initialization

I had rearranged the diagram since I expected 2493 to be merged earlier than Mehdi's branch. I can reverse my changes, except the y(start = ...).

Reply:
Ok, thanks. Yes, that would be the best solution, thanks